features/host.feature

```diff
@@ -1,20 +1,50 @@
-Feature: Display Host object fields.
+Feature: Host object
 
-  Background:
+  Scenario: API key of a newly created host is available and valid
     Given a new host
-
-  Scenario: API key of a newly created host is available and valid.
-    Then I run the code:
+    Then I can run the code:
```

Comment:
Seems like this `run the code` is an abuse of cucumber, since the features are meant to be written in high-level language.  More idiomatic would be to push all the code into the step defintion, which would describe what the lines below are testing.

Reply:
It's not great tbh but it's the framework that's there currently for testing. I can see both that it's not great as well as why it was put there though.